client/main.lua

```diff
@@ -618,9 +650,7 @@ RegisterNetEvent('qbx_vehicleshop:client:testDrive', function(args)
     })
 
     lib.waitFor(function()
-        if NetworkDoesEntityExistWithNetworkId(netId) then
-            return true
-        end
+        return NetworkDoesEntityExistWithNetworkId(netId)
```

Comment:
I've had issues with waitFor and this. I believe it needs to return nil implicitly if the condition hasn't been met. Maybe explicitly would work as well, but the way it was done before will definitely work.